snapshot/lib/snapshot/reset_simulators.rb

```diff
@@ -18,9 +18,17 @@ def self.clear_everything!(ios_versions, force = false)
       if ios_versions
         ios_versions.each do |version|
           FastlaneCore::Simulator.delete_all_by_version(os_version: version)
+          FastlaneCore::SimulatorTV.delete_all_by_version(os_version: version)
+        end
+        if version.include?('11')
```

Comment:
Could not think of a better way to handle it, so I left it. In hope of suggestion!

Reply:
Also tvOS does not always match ios version.
Perhaps I should just create more options on the reset_simulators command?

Reply:
Well, I think it is unwarrented since again Simulator handles this better.
Delete All doesn't touch the unavailable. While the code prior to my changes did.